docs/tutorials/01_ui/viz_check_boxes.py

```diff
@@ -0,0 +1,251 @@
+    arrow.SetTipResolution(tip_resolution)
+
+    # Compute a basis
+    normalizedX = [0 for i in range(3)]
+    normalizedY = [0 for i in range(3)]
+    normalizedZ = [0 for i in range(3)]
+
+    # The X axis is a vector from start to end
+    math = window.vtk.vtkMath()
+    math.Subtract(end_point, start_point, normalizedX)
+    length = math.Norm(normalizedX)
+    math.Normalize(normalizedX)
+
+    # The Z axis is an arbitrary vector cross X
+    arbitrary = [60, 10, 0]
+    math.Cross(normalizedX, arbitrary, normalizedZ)
+    math.Normalize(normalizedZ)
+
+    # The Y axis is Z cross X
+    math.Cross(normalizedZ, normalizedX, normalizedY)
+    matrix = window.vtk.vtkMatrix4x4()
+
+    # Create the direction cosine matrix
+    matrix.Identity()
+    for i in range(3):
+        matrix.SetElement(i, 0, normalizedX[i])
+        matrix.SetElement(i, 1, normalizedY[i])
+        matrix.SetElement(i, 2, normalizedZ[i])
+
+    # Apply the transforms
+    transform = window.vtk.vtkTransform()
+    transform.Translate(start_point)
+    transform.Concatenate(matrix)
+    transform.Scale(length, length, length)
+
+    # Transform the polydata
+    transformPD = window.vtk.vtkTransformPolyDataFilter()
+    transformPD.SetTransform(transform)
+    transformPD.SetInputConnection(arrow.GetOutputPort())
+
+    # Create a mapper and actor for the arrow
+    arrow_mapper = window.vtk.vtkPolyDataMapper()
+    arrow_actor = window.vtk.vtkActor()
+    arrow_actor.SetUserMatrix(transform.GetMatrix())
+
+    if window.vtk.VTK_MAJOR_VERSION <= 5:
+        arrow_mapper.SetInput(arrow.GetOutput())
+    else:
+        arrow_mapper.SetInputConnection(arrow.GetOutputPort())
+
+    if color is not None:
+        arrow_actor.GetProperty().SetColor(color)
+
+    arrow_actor.SetMapper(arrow_mapper)
+
+    return arrow_actor
+
+
+# Get difference between two lists.
+def sym_diff(l1, l2):
+    return list(set(l1).symmetric_difference(set(l2)))
+
+
+# Set Visiblity of the figures
+def set_figure_visiblity(checkboxes):
+    checked = checkboxes.checked_labels
+    unchecked = sym_diff(list(figure_dict), checked)
+
+    for visible in checked:
+        figure_dict[visible].SetVisibility(True)
+
+    for invisible in unchecked:
+        figure_dict[invisible].SetVisibility(False)
+
+
+# Toggle colors of the figures
+def toggle_color(radio):
+    color = options[radio.checked_labels[0]]
+    for _, figure in figure_dict.items():
+        figure.GetProperty().SetColor(*color)
+
+
+cube = cube_maker(color=(0, 0, 1), size=(20, 20, 20), center=(15, 0, 0))
+sphere = sphere_maker(color=(0, 0, 1), radius=11.0, center=(50, 0, 0),
+                      theta_resolution=360, phi_resolution=360)
+cone = cone_maker(color=(0, 0, 1), radius=10.0, center=(-20, -0.5, 0),
+                  height=20)
+arrow = arrow_maker(color=(0, 0, 1), start_point=(0, 25, 0),
+                    end_point=(40, 25, 0), shaft_resolution=50,
+                    tip_resolution=50)
```

Comment:
Can you remove `cube_maker`, `sphere_make`r, `cone_maker`, `arrow_make`r and replace them by 
`actor.cone`, `actor.cube`, `actor.sphere`, `actor.arrow`.

you need to import `actor` module on the top

Reply:
That's cool, I'll have a look at it.
But then why are we not using `actor.cube` in other tutorials as well?

Reply:
Just because they are old and they need to be updated :smile:

Reply:
cool, I was looking at the `actor` earlier while coding the tutorials and was wondering why are we not incorporating this instead.
In this case, I should it's the `viz_ui.py` and `viz_ui_slider.py` as well, maybe in a separate PR.

Reply:
@skoudoro I'm unable to toggle colors of the figures when using `actor.cube`.
I've checked the value is being set(using cube.GetProperty().GetColor()) but it's not reflecting on the window.
what could be the reason? maybe the actors which are returned are the reason?

Reply:
Indeed, it is much more complicated to update the colors. We need to create some functions/interfaces to make it easier. So, keep it as it is for the moment and just do the other corrections.
To get the color, you can use `cube.GetMapper().GetColorMapColors()` but it is not convenient to update it. It needs some thinking.
Thank you @PaulNicolasHunter !

Reply:
Alright, I understand @skoudoro, I'll try to dig more into it and find a workaround as soon as possible.